elements/pl-parsons-problem/pl-parsons-problem.js

```diff
@@ -0,0 +1,192 @@
+// change
+
+//change
+var parsons = {
+    options: { can_indent: true, x_indent: 40 },
+    indents: {},
+//     lines: [],
+    answer_lines: '-',
+//     answer_contents: '',
+};
+
+parsons.initialize_indentation = function() {
+    /* globals $ */
+    var submitted = $('#parsons-input').val();
+    if (submitted) {
+        var ids = submitted.split('-');
+         /* globals _ */
+        _.each(ids, function(encoded) {
+            var disaggregated = encoded.split(':');
+            var id = disaggregated[0];
+            var indent = disaggregated[1];
+            parsons.indents[id] = indent;
+            $('#' + id).addClass('indent' + indent);
+        });
+    }
+};
+
+parsons.remove_indent_classes = function(id) {
+    parsons.indents[id] = 0;
+    $('#' + id).removeClass('indent0 indent1 indent2 indent3 indent4 correct incorrect misaligned');
+};
+
+parsons.set_answer = function() {
+    var dom_objs = $('#ul-answer').children();
+    var ids = _.map(dom_objs, function(obj) { return obj.id + ':' + parsons.indents[obj.id]; }); 
+    this.answer_lines = ids ? ids.join('-') : '-';
+    console.log(this.answer_lines);
+    $('#parsons-input').val(this.answer_lines);
+    
+//     var contents = '';
+//     _.each(dom_objs, function(obj) {
+//         var lineObject = parsons.getLineById(obj.id);
+//         _.times(lineObject.indent, function() { contents += '  '; });
+//         contents += lineObject.code + '\n';
+//     });
+//     this.answer_contents = contents;
+//     console.log(contents);
+
+//     this.set_answer();
+};
+
+// parsons.getLineById = function(id) {
+//     return _.find(this.lines, { 'id': id });
+// };
+
+// parsons.clearFeedback = function(sortableId) {
+//     return;
+//     
+// //       if (this.feedback_exists) {
+// //           $("#ul-" + sortableId).removeClass("incorrect correct");
+// //           var li_elements = $("#ul-" + sortableId + " li");
+// //           $.each(this.FEEDBACK_STYLES, function(index, value) {
+// //               li_elements.removeClass(value);
+// //           });
+// //       }
+// //       this.feedback_exists = false;
+// };
+
```

Comment:
Not just here, but in general can we remove these large blocks of commented out code?

```suggestion
```